models/wrf_hydro/model_mod.f90

```diff
@@ -110,17 +111,14 @@ module model_mod
 
 logical, save :: module_initialized = .false.
 
+logical, parameter :: use_clamping = .true.
+
 character(len=512) :: string1, string2, string3
 
 integer(i8) :: model_size
 type(time_type) :: time_step
 
-! Codes for interpreting the columns of the variable_table
-integer, parameter :: VT_VARNAMEINDX  = 1 ! ... variable name
-integer, parameter :: VT_KINDINDX     = 2 ! ... DART kind
-integer, parameter :: VT_MINVALINDX   = 3 ! ... minimum value if any
-integer, parameter :: VT_MAXVALINDX   = 4 ! ... maximum value if any
-integer, parameter :: VT_STATEINDX    = 5 ! ... update (state) or not
+! For interpreting the columns of the variable_table
```

Comment:
Not sure this comment is true anymore, it is just M*N no info about which column is which